src/cf-atomic-component/src/utilities/data-set/index.js

```diff
@@ -0,0 +1,53 @@
+/* ==========================================================================
+   Data set
+
+   Utility for retrieving HTMLElement.dataset.
+
+   ========================================================================== */
+
+'use strict';
+
+const _assign = require( '../object-assign' ).assign;
+
+/**
+ * Converts a string from selector-case to camelCase.
+ * @param {string} str - The string in selector-case form.
+ * @returns {string} The string in camelCase form.
+ */
+function _toCamelCase( str ) {
```

Comment:
Should we move this into it's own utility?

Reply:
Prolly best to leave structural changes as a patch after it's released so it's easier to track the changes from the atomic-component repo to here.

Reply:
yep yep, makes sense.